File: 2813.cpp
#include <bits/stdc++.h>
using namespace std;

int main(){
    int temt = 0, temc = 0, t=0,c = 0,n;
    string ida,volta;
    cin>>n;
    for(int i=0;i<n;i++){
        cin>>ida;
        cin>>volta;
        if(ida=="chuva"){
            if(temc==0) c++;
            else temc--;
            temt++;
        }
        if(volta=="chuva"){
            if(temt==0)t++;
            else temt--;
            temc++;
        }
    }
    cout<<c<<" "<<t<<endl;
}  

File: 2813.py
def main():
    t = 0
    c = 0
    temt = 0
    temc = 0
    n = int(input())
    for i in range(0,n):
        tex = input()
        lista = tex.split()
        if(lista[0]=="chuva"):
            if(temc==0):
                c+=1
            else:
                temc-=1
            temt+=1
        if(lista[1]=="chuva"):
            if(temt ==0):
                t+=1
            else:
                temt-=1
            temc+=1
        del lista[:]
    print(c, t)
main()